mkt/developers/tasks.py

```diff
@@ -63,17 +64,15 @@ def validator(upload_id, **kw):
         return
 
     temp_path = None
-    # Make a copy of the file if it's a packaged app since we can't assume the
+    # Make a copy of the file since we can't assume the
     # uploaded file is on the local filesystem.
-    if upload.name.endswith('.zip'):
```

Comment:
Nevermind, it does require the contents of the manifest. How did tests pass with that change then ?